### ISSUE TITLE
`Changes are too large to display` 

### ISSUE THREAD

**Issue by marwie** [ForkIssues/TrackerWin#2012]
Using fork 1.89.0.0 on win 10

displaying:

![image](https://github.com/fork-dev/TrackerWin/assets/5083203/17d8719f-e222-45e7-85f7-9f2d8a228172)

for these changes

![image](https://github.com/fork-dev/TrackerWin/assets/5083203/44fbc586-77a0-4a28-b5bd-ec81818eeddd)

https://github.com/fork-dev/TrackerWin/assets/5083203/62679b45-9d2e-4ae6-9f24-b04715256ca2

**Comment by DanPristupov** [comment]
I don't see a question.

**Comment by marwie** [comment]
Is it expected that a change of not quite 50 lines is too large to display?

**Comment by DanPristupov** [comment]
> Is it expected that a change of not quite 50 lines is too large to display?

Yes, because number of lines is irrelevant. Minimizing a file to one-line does not make the changes smaller.

The tiny horizontal scrollbar on your video shows that the changes are large.

**Comment by robertsynoradzki** [comment]
I also encountered the same thing while I was displaying changes to a file that was less than 200 bytes in size that contained about 15 lines ~30 characters each, and the difference was in one or two lines. Don't remember the facts, but it was some sort of `appsettings.json`, `.editorconfig` or `.gitignore`. "Changes are too large to display" was truly and surely **not** the real problem, because:
1. The content was sincerely small, as I've described.
2. I've been working on the file for a year now and never before I have encountered this. It happened several weeks ago after one of the updates (I always update ASAP).

Unfortunately I do not have a reproduction now, but there is something happening here for sure.

**Comment by adams13x13** [comment]
Similar issue: #496

**Comment by Leonardo76** [comment]
Change the text control.
